### PR TITLE
Create MeetingsSequenceDiagram

### LINKED ISSUE #86: Create MeetingsSequenceDiagram

### PR DESCRIPTION
**Pull request by XinghuaJulia**
Fixes  #86

### CODE REVIEW

File: docs/diagrams/MeetingsSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@startuml
```
Comment (psh12320) [inline docs/diagrams/MeetingsSequenceDiagram.puml:1]: Everything LGTM!

**Review by psh12320 — COMMENTED** [review]
LGTM!

File: docs/diagrams/MeetingsSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+CommandResult --> MeetingsCommand
+deactivate CommandResult
+
+MeetingsCommand --> LogicManager : r
+deactivate MeetingsCommand
+
+[<--LogicManager
+deactivate LogicManager
+@enduml
```
Comment (marraknas) [inline docs/diagrams/MeetingsSequenceDiagram.puml:53]: Diagram is good!

**Review by marraknas — COMMENTED** [review]
LGTM!

File: docs/diagrams/MeetingsSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "m:MeetingsCommand" as MeetingsCommand LOGIC_COLOR
+participant "r:CommandResult" as CommandResult LOGIC_COLOR
+end box
```
Comment (PeterHW963) [inline docs/diagrams/MeetingsSequenceDiagram.puml:10]: the diagram is so good I'm getting turned o- 😋

**Review by PeterHW963 — COMMENTED** [review]
LGTM!

**Review by ulricolo7 — COMMENTED** [review]
LGTM!

**Pull request merged**